src/Compilers/CSharp/Portable/BoundTree/BoundNodes.xml

```diff
@@ -2022,9 +2022,13 @@
     <Field Name="Format" Type="BoundLiteral?"/>
   </Node>
   
+    <!-- An 'is pattern'. The fields DecisionDag, WhenTrueLabel, and WhenFalseLabel represent the inner pattern
+         after removing any outer 'not's, so consumers (such as lowering and definite assignment of the local in
+         'is not Type t') will need to compensate for negated patterns. -->
   <Node Name="BoundIsPatternExpression" Base="BoundExpression">
     <Field Name="Expression" Type="BoundExpression" Null="disallow"/>
     <Field Name="Pattern" Type="BoundPattern" Null="disallow"/>
+    <Field Name="IsNegated" Type="bool"/>
```

Comment:
>IsNegated [](start = 17, length = 9)

I think this property needs a comment, or, alternatively, should be mentioned in the comment above.